pkg/cmd/listen_test.go

```diff
@@ -3,7 +3,7 @@ package cmd
 import (
 	"testing"
 
-	assert "github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/assert"
```

Comment:
Not sure what the general project convention is, but FYI in case you didn't know: there's a slight difference between assert and require in that the require variant will fail tests on the first failure. Generally I've found this to be a sane default personally, but admittedly I probably deviate from the Go community in that opinion.

Reply:
Oh, I was mostly fixing the inconsistency because some tests used `assert` and others aliased `assert` to `require`.

I've updated everything to use `require` directly.